programs/eosio-launcher/main.cpp

```diff
@@ -500,7 +501,8 @@ launcher_def::set_options (bpo::options_description &cfg) {
     ("gelf-endpoint",bpo::value<string>(&gelf_endpoint)->default_value("10.160.11.21:12201"),"hostname:port or ip:port of GELF endpoint")
     ("template",bpo::value<string>(&start_temp)->default_value("testnet.template"),"the startup script template")
     ("script",bpo::value<string>(&start_script)->default_value("bios_boot.sh"),"the generated startup script name")
-    ("max-block-cpu-usage",bpo::value<uint32_t>(),"")
+    ("max-block-cpu-usage",bpo::value<uint32_t>(),"Provide the \"max-block-cpu-usage\" value to use in the genesis.json file")
```

Comment:
The parameters are optional<uint32_t>s, which get a lexical cast error.  To use that idiom, it would have to use uint32_t and use 0 as the not used value, but I think the current implementation is cleaner than that